src/testing/run.ts

```diff
@@ -143,18 +126,7 @@ async function runEvaluator<TestCaseType, OutputType>(args: {
       evaluatorId: args.evaluator.id,
       error: err,
     });
-    return;
   }
-
-  if (evaluation === undefined) return;
-
-  await client.post('/evals', {
-    testExternalId: args.testId,
-    testCaseHash: args.testCaseHash,
-    evaluatorExternalId: args.evaluator.id,
-    score: evaluation.score,
-    threshold: evaluation.threshold,
-  });
```

Comment:
fyi - I have this line updated in my PR to include metadata

Reply:
yep np, i'll resolve when the merge conflicts pops up